### PR TITLE
input.conf: add comments

### PR DESCRIPTION
**Pull request by guidocella**
These will be shown in the new keybinding list.

### CODE REVIEW

**Comment by avih** [comment]
I'm leaving reviewing the language to others

But do keep in mind that in this context - simple and short descriptions are better than long ones.

File: etc/input.conf [hunk 1]
```diff
@@ -29,163 +29,152 @@
...
+#Alt+up    add video-pan-y  0.1         # move the video up
+#Alt+down  add video-pan-y -0.1         # move the video down
+#Alt++     add video-zoom   0.1         # zoom in
+#Alt+-     add video-zoom  -0.1         # zoom out
+#Alt+BS set video-zoom 0 ; set video-pan-x 0 ; set video-pan-y 0 # reset zoom and pan settings
+#PGUP add chapter 1                     # skip to the next chapter
+#PGDWN add chapter -1                   # skip to the previous chapter
+#Shift+PGUP seek 600                    # seek 10 minutes forward
+#Shift+PGDWN seek -600                  # seek 10 minutes backward
+#[ multiply speed 1/1.1                 # decrease the speed
```
Comment (occivink) [inline etc/input.conf:71]: mention 'playback speed' maybe?

File: etc/input.conf [hunk 2]
```diff
@@ -29,163 +29,152 @@
...
-#ESC {encode} quit 4
+#Shift+RIGHT no-osd seek  1 exact       # seek exactly 1 second forward
+#Shift+LEFT  no-osd seek -1 exact       # seek exactly 1 second backward
+#Shift+UP    no-osd seek  5 exact       # seek exactly 5 seconds forward
+#Shift+DOWN  no-osd seek -5 exact       # seek exactly 5 seconds backward
+#Ctrl+LEFT   no-osd sub-seek -1         # skip to the previous subtitle
+#Ctrl+RIGHT  no-osd sub-seek  1         # skip to the next subtitle
+#Ctrl+Shift+LEFT sub-step -1            # change subtitle timing such that the previous subtitle is displayed
+#Ctrl+Shift+RIGHT sub-step 1            # change subtitle timing such that the next subtitle is displayed
+#Alt+left  add video-pan-x  0.1         # move the video left
```
Comment (occivink) [inline etc/input.conf:60]: technically the video is moving right when pressing 'alt+left'. The binding still makes if you think of it as the camera/viewpoint moving left

**Comment by hooke007** [comment]
Is it really necessary to make every shortcut has comments?
which look a little bit mess.

**Comment by guidocella** [comment]
Rebased, though it's a pity that `set current-window-scale` is now grouped under `current` instead of `window`.

**Comment by avih** [comment]
> it's a pity that set current-window-scale is now grouped under current instead of window.

Very easy to take care of (add `current` to `name_prefixes` at `stats.lua`).

**Comment by avih** [comment]
Thanks.

In the future we could refine it further, maybe removing trivial/obvious comments and/or shortening some comments. This is not set in stone.

**Pull request merged**